src/components/Orders/Orders.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import Order from './Order';
+
+class Orders extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            menu: {},
+            deliveryPrice: 2.50
```

Comment:
Since delivery price does not change is can go into a variable rather than be in state. For example put it on line 3 as `const DELIVERY_PRICE = 2.50`